test/ledger.js

```diff
@@ -0,0 +1,123 @@
+var async       = require("async");
+var assert      = require('assert');
+var lodash       = require('lodash');
+var Amount      = require("ripple-lib").Amount;
+var Remote      = require("ripple-lib").Remote;
+var Transaction = require("ripple-lib").Transaction;
+var Server      = require("./server").Server;
+var testutils   = require("./testutils");
+var config      = testutils.init_config();
+
+suite('Ledger requests', function() {
+  var $ = { };
+  var expectedledgers = [
+    {"ledger":{"accepted":true,"account_hash":"A21ED30C04C88046FC61DB9DC19375EEDBD365FD8C17286F27127DF804E9CAA6","closed":true,"ledger_index":"1","seqNum":"1","totalCoins":"100000000000000000","total_coins":"100000000000000000","transaction_hash":"0000000000000000000000000000000000000000000000000000000000000000"},"ledger_index":1,"validated":true},
```

Comment:
Those lines are a raw dump of the ledger request that I just copied and pasted and left as-is, because lazy. I'll clean up the formatting.